src/index.js

```diff
@@ -2,21 +2,21 @@ import md5 from 'md5';
 import { Base64 } from 'js-base64';
 
 import {
-  VERSION,
-  DOMAIN_REGEX,
-  DEFAULT_OPTIONS,
-  DPR_QUALITIES,
-  DPR_TARGETS_RATIOS,
+    VERSION,
+    DOMAIN_REGEX,
+    DEFAULT_OPTIONS,
+    DPR_QUALITIES,
+    DEFAULT_DPR,
```

Comment:
Just checking, are these extra spaces intentional?

Reply:
My bad, probably webstorm and prettier don't pass after.

Reply:
I resolved this and pushed to this PR 👍